addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -110,6 +112,8 @@ public static Commands fromByte(int input) throws RFXComUnsupportedValueExceptio
     public byte signalLevel;
     public boolean group;
 
+    private static byte[] lastUnit = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
This state is specific to the X10 protocol and the functionality of those X10 controllers that possess bright/dim buttons. It should not be exposed outside Lighting1. It is correct that it is never reset (strictly speaking it resets when the dimmers owning the state get power cycled but X10 dimmers are receive only so there's no way to know if that happens). It doesn't _need_ to be any more complicated than it is but could certainly be made into some external entity that could hook in to persistence to save state. But X10 is lossy. X10 users know that. Only too well!